### ISSUE TITLE
Calendar: Color axis uses only last two colors specified.

### ISSUE THREAD

**Issue by heoga** [google/google-visualization-issues#2197]
Please see the following jsfiddle for an example:
[https://jsfiddle.net/Heoga/d05hprmz/](https://jsfiddle.net/Heoga/d05hprmz/)

**Comment by nbering** [comment]
@heoga That's not strictly true. Try changing one of your data values to a negative number.

You might note that the documentation never shows any more than 3 colors, and the coloraxis will use 3 values, if there is a negative value present in the data. Otherwise it only renders the midpoint and high point.

**Comment by grabks** [comment]
If you explicitly provide a `values` array, it will correlate your colors to those values. Here is a jsfiddle example: https://jsfiddle.net/f0v0znyf/

Closing as "working as intended".

**Comment by heoga** [comment]
I beg to differ.  The doc states:
'You must have at least two values; the gradient will include all your
values, plus calculated intermediary values, with the first color as the
smallest value, and the last color as the highest.'
That clearly suggests it can take more than two. It also says the gradient
will include all your values.

It might be working as intended, but if that's the case your doc needs
amending.

**Comment by grabks** [comment]
You're right. At the very least, there is a difference here between how Calendar handles this option (when you don't specify `values`) and how the GeoChart handles this option. We should be more consistent.
